db/models.js

```diff
@@ -1,46 +1,55 @@
-const sequelize = require('./index.js');
+const ORM = require('./index.js');
 
-// will implement use of the authorId variable
-// let authorId = '';
-
-exports.getAuthorInfo = (bookId, callback) => {
-  const authorQuery = `SELECT name, followers, biography FROM authors WHERE id IN (SELECT author_id FROM books WHERE id = ${bookId})`;
-  sequelize.sequelize.query(authorQuery)
-    .then(([results]) => {
-      // set authorId (from line 3) to be passed into getFiveBooks
-      callback(null, results[0]);
-    })
-    .catch((err) => {
-      console.log('err', err);
-    });
-};
-
-exports.getFiveBooks = (authorId, callback) => {
+const getFiveBooks = (authorId, callback) => {
   console.log('authorId', authorId);
-  const fiveBooksQuery = `SELECT title FROM books WHERE author_id = ${authorId} ORDER BY average_ratings LIMIT 5`;
-  sequelize.sequelize.query(fiveBooksQuery)
+  const fiveBooksQuery = `SELECT title FROM books WHERE author_id = ${authorId} ORDER BY average_rating LIMIT 5`;
+  ORM.sequelize.query(fiveBooksQuery)
     .then(([results]) => {
+      // separate out as its own function, then able to test its output
       const fiveBooks = {};
       fiveBooks.titles = [];
       for (let i = 0; i < results.length; i += 1) {
-        console.log(results[i]);
         fiveBooks.titles.push(results[i].title);
       }
       callback(null, fiveBooks);
     })
     .catch((err) => {
       console.log('err', err);
+      callback(err);
     });
 };
 
-exports.getBookItemHoverWindow = (bookId, callback) => {
+const getAuthorInfo = (bookId, callback) => {
+  const authorQuery = `SELECT id, name, followers, biography FROM authors WHERE id IN (SELECT author_id FROM books WHERE id = ${bookId})`;
```

Comment:
I'd recommend making the query for this and the books `JOIN` queries, as this should let you return all the data you need in each call